src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -186,28 +212,45 @@ def get(self, request: Request, project: Project) -> Response:
                     outcome = munging_outcome
                     scope.set_tag("stacktrace_link.munged", True)
 
-                current_config = {"config": serialize(config, request.user), "outcome": outcome}
-                matched_code_mappings.append(current_config)
-                # use the provider key to be able to split up stacktrace
-                # link metrics by integration type
-                provider = current_config["config"]["provider"]["key"]
-                scope.set_tag("integration_provider", provider)  # e.g. github
+                current_config = {
+                    "config": serialize(config, request.user),
+                    "outcome": outcome,
+                    "automatically_generated": config.automatically_generated,
```

Comment:
Add `automatically_generated` so we can tell which code mappings are user or Sentry generated when determining precedence